app/assets/javascripts/application.js

```diff
@@ -28,6 +28,8 @@
 // GOV.UK Design System
 //= require all.js
 
+document.getElementsByTagName('html')[0].className += 'js'
```

Comment:
what is this change doing?

Reply:
By adding a `js` class to the html element, I can hide the submit button on the registers collection page 'Sort by' form using CSS - this means the button doesn't flash in and out of view for those with JavaScript enabled and it shows instantly when people have JavaScript disabled.

Reply:
Can you add a comment for it? I think it makes sense but I wouldn't have understood it without that explanation.

Reply:
✍️ done